src/ktn_code.erl

```diff
@@ -26,15 +27,29 @@
       attrs => map(),
       node_attrs => map(),
       content => [tree_node()]}.
+-type beam_lib_beam() :: file:filename() | binary().
+% Should eventually become beam_lib:beam(), once that's exposed.
+-type erl_syntax_annotation_or_location() :: erl_anno:anno() | erl_anno:location().
+% Should eventually become erl_syntax:annotation_or_location(), once that's exposed.
```

Comment:
Already pull requested for this: https://github.com/erlang/otp/pull/7535

Reply:
Maybe put the link to the PR in the comment itself.

Reply:
55df382